package.json

```diff
@@ -30,35 +30,41 @@
     "onCommand:vscode-ansible.playbook-in-docker",
     "onCommand:vscode-ansible.playbook-in-localansible",
     "onCommand:vscode-ansible.ssh",
-    "onCommand:vscode-ansible.deployment-templates"
+    "onCommand:vscode-ansible.deployment-templates",
+    "onCommand:vscode-ansible.sync-folder-ssh"
   ],
   "main": "./out/extension",
   "contributes": {
     "commands": [
       {
         "command": "vscode-ansible.cloudshell",
         "title": "Run Ansible Playbook in Cloud Shell",
-        "category": "ansible"
+        "category": "Ansible"
       },
       {
         "command": "vscode-ansible.playbook-in-docker",
         "title": "Run Ansible Playbook in Docker",
-        "category": "ansible"
+        "category": "Ansible"
       },
       {
         "command": "vscode-ansible.playbook-in-localansible",
         "title": "Run Ansible Playbook in Local Ansible",
-        "category": "ansible"
+        "category": "Ansible"
       },
       {
         "command": "vscode-ansible.ssh",
         "title": "Run Ansible Playbook Remotely via ssh",
-        "category": "ansible"
+        "category": "Ansible"
       },
```

Comment:
probably these changes should be in a separate PR, but that's ok, as they are already there